### PR TITLE
Add a lot of MapEntity Settings

### PR DESCRIPTION
**Pull request by krypciak**

### CODE REVIEW

File: modules/impact.base.actor-entity.d.ts [hunk 1]
```diff
@@ -5,6 +5,47 @@ export {};
...
+        brake?: string;
+        preIdle?: string;
+        jump?: string;
+        fall?: string;
+        hover?: string;
+        preHoverMove?: string;
+        hoverMove?: string;
+        hoverMoveRev?: string;
+        land?: string;
+      }
```
Comment (krypciak) [inline modules/impact.base.actor-entity.d.ts]: this is accidentally stolen from EL's branch please don't sue me

File: modules/game.feature.menu.map-model.d.ts [hunk 2]
```diff
@@ -45,8 +45,8 @@ declare global {
 
       namespace MapEntity {
         interface Settings {
-          mapId?: number,
-          name?: string,
+          mapId: number,
+          name: string,
```
Comment (elluminance) [inline modules/game.feature.menu.map-model.d.ts]: `name` is not a required property for entities.

**Comment by krypciak** [comment]
I checked every type I implemented and all of them have their proper settings in proper order

File: modules/game.feature.menu.map-model.d.ts (hunk 2 above)
Comment (elluminance) [reply modules/game.feature.menu.map-model.d.ts]: Oops, I accidentally selected an old commit for commenting on. I noticed this is still the case in the most recent version, however.

**Review by lexisother — APPROVED** [review]

**Pull request merged**